src/app.ts

```diff
@@ -167,6 +168,25 @@ export default (keycloak: Keycloak, getProject: (projectId: string) => ArrangerP
         }
     });
 
+    app.post('/sets/aliases', keycloak.protect(), async (req, res, next) => {
+        const isPlainObject = (input: unknown) => Object.prototype.toString.call(input) === '[object Object]';
+        try {
+            const queries = req.body?.queries;
+            if (!queries || !Array.isArray(queries) || queries.some(q => !isPlainObject(q))) {
```

Comment:
I would let users-api validating the payload and keep arranger as a simple pass-through